src/application/application.js

```diff
@@ -16,12 +14,6 @@ export default Application.extend({
     this.$body = $(document.body);
     this.layout = new LayoutView();
     this.layout.render();
-
-    this.listenTo(routerChannel, {
-      'before:enter:route' : this.onBeforeEnterRoute,
-      'enter:route'        : this.onEnterRoute,
-      'error:route'        : this.onErrorRoute
-    });
```

Comment:
I wouldn't be opposed to modifying https://github.com/thejameskyle/backbone-routing to forward events to `Backbone.history`.